cmd/kops/update_cluster.go

```diff
@@ -206,6 +200,34 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		}
 	}
 
+	lifecycleOverrideMap := make(map[cloudup.Phase]fi.Lifecycle)
+
+	for _, override := range c.PhaseOverrides {
+
+		values := strings.Split(override, "=")
+		phaseName := values[0]
+		lifecycle := values[1]
+
+		phaseOverride, err := getPhase(phaseName)
+		if err != nil {
+			return fmt.Errorf("incorrect phase name in overides: %v", err)
+		}
+
+		switch lifecycle {
+		case string(fi.LifecycleExistsAndValidates):
+			lifecycleOverrideMap[phaseOverride] = fi.LifecycleExistsAndValidates
+		case string(fi.LifecycleIgnore):
+			lifecycleOverrideMap[phaseOverride] = fi.LifecycleIgnore
+		case string(fi.LifecycleWarnIfInsufficientAccess):
+			lifecycleOverrideMap[phaseOverride] = fi.LifecycleWarnIfInsufficientAccess
+		case string(fi.LifecycleExistsAndWarnIfChanges):
+			lifecycleOverrideMap[phaseOverride] = fi.LifecycleExistsAndWarnIfChanges
+		default:
+			return fmt.Errorf("overrides error, unknown lifecycle %q, available lifecycles: %s", lifecycle, strings.Join(fi.Lifecycles.List(), ","))
```

Comment:
`unknown lifecycle %q in overrides, available lifecycles: %s`

Reply:
Or, could we just have a parse function instead?  In particular because we can call `fi.Lifecycles.List()` to avoid the switch

Reply:
The err message is populating from the new func.